src/pyskindose/phantom_class.py

```diff
@@ -289,14 +291,20 @@ def __init__(self,
                       0.5, 0.25, 0.25, -0.25, -0.25, -0.5, -0.5, 0.5]]
 
             # Lateral position of the the vertices
-            y_pad = [index * phantom_dim.pad_length for index in
-                     [1.0, 1.0, 1, 1, 1.0, 1.0, 0, 0,
-                      1.0, 1.0, 1, 1, 1.0, 1.0, 0, 0]]
+            y_pad = [index * phantom_dim.pad_thickness for index in
+                     [0, 0, 0, 0, 0, 0, 0, 0, -1, -1, -1, -1, -1, -1, -1, -1]]
 
             # Vertical position of the vertices
-            z_pad = [index * phantom_dim.pad_thickness for index in
-                     [0, 0, 0, 0, 0, 0, 0, 0,
-                      1, 1, 1, 1, 1, 1, 1, 1]]
+            z_pad = [index * phantom_dim.pad_length for index in
+                     [0, 0, 0, 0, 0, 0, -1, -1, 0, 0, 0, 0, 0, 0, -1, -1]]
+
```

Comment:
This seems to be kind of a duplicate of what's happening at row 256-261. Could we break it out to not have the same code repeating?

Reply:
This has been improved in 2e0c900c95f5c1c11a2e4f67cb162cdd8f585edc